src/Features/CSharp/Portable/EditAndContinue/SyntaxComparer.cs

```diff
@@ -310,6 +312,17 @@ private static Label ClassifyStatementSyntax(SyntaxKind kind, SyntaxNode? node,
                 // Expressions are ignored but they may contain nodes that should be matched by tree comparer.
                 // (e.g. lambdas, declaration expressions). Descending to these nodes is handled in EnumerateChildren.
 
+                case SyntaxKind.NamespaceDeclaration:
```

Comment:
This will have to be updated when I merge this change into the `features/FileScopedNamespaces` branch. Just curious, what is the consequence of failing to update this? Is it in perf or correctness?

Reply:
There is a checklist for EnC for new language features that Tomas put together, I'll forward it to you.

In general, if a node isn't classified in these switch statements it means its invisible to EnC, so for example for file scoped namespaces we wouldn't report a rude edit if one was changed or added.